fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzeStmtAnalyzer.java

```diff
@@ -167,7 +167,8 @@ private void analyzeAnalyzeTypeDesc(ConnectContext session, AnalyzeStmt statemen
                     Column column = analyzeTable.getColumn(columnName);
                     if (column.getType().isComplexType()
                             || column.getType().isJsonType()
-                            || column.getType().isOnlyMetricType()) {
+                            || column.getType().isOnlyMetricType()
+                            || column.getType().isChar() || column.getType().isVarchar()) {
```

Comment:
Why varchar couldn't support histogram?

Reply:
Because our statistical information calculation formulas currently convert the value to double for comparison.
`    public static Statistics estimateColumnToConstantComparison(Optional<ColumnRefOperator> columnRefOperator,
                                                                ColumnStatistic columnStatistic,
                                                                BinaryPredicateOperator predicate,
                                                                OptionalDouble constant,
                                                                Statistics statistics)`

The framework of statistical information estimation needs to be modified. This is the next step. Before this step is completed, the string type should be disabled.